src/__tests__/suggestions.js

```diff
@@ -74,7 +74,9 @@ test(`should not suggest if the suggestion would give different results`, () =>
 })
 
 test('should suggest by label over title', () => {
-  renderIntoDocument(`<label><span>bar</span><input title="foo" /></label>`)
+  renderIntoDocument(
+    `<label><span>bar</span><input type="password" title="foo" /></label>`,
```

Comment:
To be accurate: The spec related to aria didn't change. This was a bug in earlier versions of `aria-query`. `<input />` with no or an invalid `type` attribute is always considered `<input type="text" />` which is a `role="textbox"`.